src/site/content/en/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -37,7 +41,7 @@ Soon after [@vjeux](https://twitter.com/vjeux) created the initial version of Ex
 January 2020 and [blogged about it](reflections-on-excalidraw/), he proposed the following in
 [Issue #561](https://github.com/excalidraw/excalidraw/issues/561#issue-555138343):
 
-> Would be great to wrap Excalidraw within Electron (or equivalent) and publish it as a native
+> Would be great to wrap Excalidraw within Electron (or equivalent) and publish it as a [platform-specific]
```

Comment:
Making visible that I have altered the quote without changing its sense. This looks like a great solution.

Reply:
I like that approach